pypesto/visualize/profiles.py

```diff
@@ -276,9 +287,11 @@ def handle_inputs(result, profile_indices):
     fvals = []
     for i_par in range(0, len(result.profile_result.list[0])):
         if i_par in profile_indices:
-            tmp = np.array(
-                [result.profile_result.list[0][i_par].x_path[i_par, :],
-                 result.profile_result.list[0][i_par].ratio_path[:]])
+            tmp = np.array([result.profile_result.list[profile_list][
```

Comment:
sorry, but this code looks sooo autoformatted :)

Reply:
yes, arbitrarily ugly...
will change that...